tekton/pipeline_run_test.go

```diff
@@ -210,8 +210,15 @@ var _ = Describe("PipelineRun", func() {
 
 		It("can add the ReleaseStrategy information to a PipelineRun object and ", func() {
```

Comment:
Let's fix the title as part of this change. Also, we need a test for both scenarios:

- Pipeline is present and bundle is not.
- Both pipeline and bundle are present.

Reply:
Thanks, Done! :)

Reply:
@davidmogar, as we verified the test offline, you were Happy. I am resolving this comment.